evalml/pipelines/components/estimators/regressors/arima_regressor.py

```diff
@@ -226,7 +246,17 @@ def predict(self, X, y=None):
             },
         )
         if not X.empty and self.use_covariates:
-            y_pred = self._component_obj.predict(fh=fh_, X=X)
+            if fh_[0] != 1:
+                # pmdarima (which sktime uses under the hood) only forecasts off the training data
+                # but sktime circumvents this by predicting everything from the end of training data to the date / periods requested
+                # and only returning the values for dates / periods given to sktime. Because of this,
+                # pmdarima requires the number of covariate rows to equal the length of the total number of periods (X.shape[0] == fh_[-1]) if covariates are used.
+                # We circument this by adding arbitrary rows to the start of X since sktime discards these values when predicting.
```

Comment:
```suggestion
                # We circumvent this by adding arbitrary rows to the start of X since sktime discards these values when predicting.
```